packages/store/addon/-private/store-service.ts

```diff
@@ -134,20 +134,20 @@ export interface CreateRecordProperties {
   The store provides multiple ways to create new record objects. They have
   some subtle differences in their use which are detailed below:
 
-  [createRecord](../methods/createRecord?anchor=createRecord) is used for creating new
+  [createRecord](../../classes/Adapter/methods?anchor=createRecord) is used for creating new
```

Comment:
Adapter is the wrong class, probably want Store here

Reply:
more likely just need to do `./` instead of `../`

Reply:
Adapter -> Store ✔️ do agree.

Page where the link is found:

```
https://api.emberjs.com/ember-data/4.7/modules/@ember-data%2Fstore
```

Page where the user should land:
```
https://api.emberjs.com/ember-data/4.7/classes/Store/methods?anchor=createRecord
```

That's why I did `..` twice as it seems like the link should get out of `@ember-data/store` and then out of `modules` first.

So my conclusion here is that `./` is relative to `https://api.emberjs.com/ember-data/4.7/`?

Reply:
Pushed fix.

Reply:
👍